flink-contrib/flink-storm-examples/src/test/java/org/apache/flink/storm/tests/operators/VerifyMetaDataBolt.java

```diff
@@ -28,6 +27,11 @@
 import org.apache.storm.tuple.Tuple;
 import org.apache.storm.tuple.Values;
 
+import java.util.Map;
+
+/**
+ * A Bolt implementation that verifies meta data emitted by a {@link MetaDataSpout}.
```

Comment:
I think that this should be "metadata" but see that the class is camelcased `MetaData`.